gdcm_wrapper.cc

```diff
@@ -41,8 +43,9 @@ struct PixelData c_decode_single_frame_compressed(
 
     struct PixelData outputStruct;
     size_t length = image.GetBufferLength();
-    outputStruct.buffer = (char*)malloc(length);
-    if (!image.GetBuffer(outputStruct.buffer)) {
+    outputStruct.buffer = (char *)malloc(length);
```

Comment:
I wonder where this piece of memory is freed. 🤔

Reply:
Good question.
`slice::from_raw_parts_mut` mentions that the memory pointed it is not to be touched as long as the ref is alive (i.e. slice is around so I guess freeing right after from_raw_parts_mut is run is not possible.

Reply:
please see [3a41dce](https://github.com/pevers/gdcm-rs/pull/7/commits/3a41dcea9a290aafef31274f66d4b859e01a182e) for a suggested implementation